### PR TITLE
Grammatical and other small edits

### PR DESCRIPTION
**Pull request by dhimmel**
Closes https://github.com/greenelab/deep-review/pull/636 by @qiyanjun. Attempts to replay those changes as much as possible on the current repo state.

Cherry-pick https://github.com/greenelab/deep-review/pull/636/commits/88eb8f3401fec014c2089773f8c514b610e7f299. Manually resolve conflicts (@dhimmel)

Manually take some parts of https://github.com/greenelab/deep-review/pull/636/commits/3adde57fb93c9bef6d094156ddeafc4e40710f2d although many changes from this commeit were already made.

Do not include the translocation of the  "Applications of deep learning to longitudinal analysis" section.

### CODE REVIEW

**Comment by cgreene** [comment]
Will the changes be ascribed to @qiyanjun or is that inconsistent with the ability to reasonably quickly get this merged into current master?

**Comment by agitter** [comment]
@cgreene did you review in full? I didn't review the original pull request.

**Comment by dhimmel** [comment]
> Will the changes be ascribed to @qiyanjun or is that inconsistent with the ability to reasonably quickly get this merged into current master?

Yes, see how she is set as the commit author.

**Comment by qiyanjun** [comment]
dhimmel, i see many changes from this commeit were already made in the current release. I recommend to just close the request!

**Comment by dhimmel** [comment]
> I recommend to just close the request!

How about we merge the changes that made it to this PR and don't worry about any parts of https://github.com/greenelab/deep-review/pull/636 that didn't make it. The changes here are changes that aren't part of master yet...

@cgreene did you review and approve? If so can you use the GitHub review menu rather than :+1:, so it's clear.

File: content/02.intro.md [hunk 1]
```diff
@@ -23,7 +23,8 @@ algorithms have produced comparable or higher accuracy than previous
 best-in-class methods that required years of extensive customization, and
 specialized implementations are now being used at industrial scales.
 
-Deep learning approaches grew from research in neural networks, which were first proposed in 1943 [@doi:10.1007/BF02478259] as a model
+Deep learning originated from the field of neural networks.
```
Comment (agitter) [inline content/02.intro.md]: This partially reverts a change that was made in #566. We'll need to do a full review before merging.  I may have time in the next 24 hrs if @cgreene doesn't do it first.

Comment (cgreene) [reply content/02.intro.md]: Currently planning to do this before 10 AM tomorrow 🤞

Comment (dhimmel) [reply content/02.intro.md]: Removed this change.

File: content/03.categorize.md [hunk 2]
```diff
@@ -50,7 +50,7 @@ focus on barriers to achieving these goals. We also highlight approaches that
 researchers are taking to address challenges within the field, particularly with
 regards to data availability and labeling.
 
-### Imaging applications in healthcare
+### Imaging based patient categorization
```
Comment (agitter) [inline content/03.categorize.md]: I prefer leaving this sub-heading as `Imaging applications in healthcare`.  Some of the applications, like anatomical localization, are not strictly about patient categorization.

File: content/03.categorize.md [hunk 3]
```diff
@@ -446,15 +446,13 @@ can be used before overfitting occurs. For example, the unsupervised and
...
-irrelevant features [@doi:10.1101/095786]. While adversarial training examples
+observations from which labels can be inferred.
+The strategies of adversarial training mentioned above can reduce overfitting, if transformations are available that preserve the meaningful content of the data while transforming irrelevant features [@doi:10.1101/095786].
+While adversarial training examples
 can be easily imagined for certain methods that operate on images, it is more
 challenging to figure out what an equivalent transformation would be for a
 patient's clinical test results. Consequently, it may be hard to employ
-adversarial training examples, not to be confused with generative adversarial
-neural networks, with other applications. Finally, approaches that transfer
+adversarial training examples with other applications. Finally, approaches that transfer
```
Comment (agitter) [inline content/03.categorize.md]: Should we add `adversarial training examples` to our new glossary table?  That would help differentiate if from GANs.

File: content/03.categorize.md [hunk 4]
```diff
@@ -690,7 +687,7 @@ Despite the difficulties and uncertainties, machine learning practitioners (and
 particularly those who use deep neural networks, which are challenging to
 interpret) must remain cognizant of these dangers and make every effort to
 prevent harm from discriminatory predictions. To reach their potential in this
-domain, deep learning methods will need to be interpretable. Researchers need to
+domain, deep learning methods will need to be interpretable (more in the discussion section). Researchers need to
```
Comment (agitter) [inline content/03.categorize.md]: Change `(more in the discussion section)` to `(see Discussion)` to be consistent with other sections.

File: content/04.study.md [hunk 5]
```diff
@@ -72,15 +72,13 @@ required data to be split randomly into two distinct models and trained
...
-power to infer gene expression. For instance, the DeepChrome CNN [@tag:Singh2016_deepchrome]
-improved prediction accuracy of high or low gene expression from histone modifications over existing
-methods. Deep learning can also integrate different data types. For example,
-Liang et al. combined RBMs to integrate gene expression, DNA methylation, and
-miRNA data to define ovarian cancer subtypes [@tag:Liang2015_exprs_cancer].
-While these approaches are promising, many convert gene expression measurements
-to categorical or binary variables, thus ablating many complex gene expression
-signatures present in intermediate and relative numbers.
+Epigenetic data, combined with deep learning, may have sufficient explanatory power to infer gene expression.
+For instance, a convolutional neural network applied to histone modifications, termed DeepChrome, improved prediction accuracy of high or low gene expression over existing non-neural network
```
Comment (agitter) [inline content/04.study.md]: This edit is reverting a change from #618.  We should keep the existing DeepChrome sentence and add the AttentiveChrome reference (changing `is` to `was`).

File: content/04.study.md [hunk 6]
```diff
@@ -501,7 +498,8 @@ transcriptional states provides unprecedented views of regulatory processes
 However, large challenges exist in studying single cells. Relatively few cells
 can be assayed at once using current droplet, imaging, or microwell
 technologies, and low-abundance molecules or modifications may not be detected
-by chance due to a phenomenon known as dropout. To solve this problem,
+by chance due to a phenomenon known as dropout, not to confused with the dropout
```
Comment (agitter) [inline content/04.study.md]: `not to be confused`

File: content/05.treat.md [hunk 7]
```diff
@@ -61,7 +61,7 @@ studies have attempted to create more interpretable deep models, either
 specifically for healthcare or as a general procedure for deep learning (see
 Discussion).
 
-#### Predicting patient trajectories
+#### Predicting patient trajectories from treatments
```
Comment (agitter) [inline content/05.treat.md]: I again prefer the existing sub-heading.  I don't know these papers well, but I suspect some of the are making predictions from general clinical data, not just treatments.

File: content/06.discussion.md [hunk 8]
```diff
@@ -219,7 +219,9 @@ focus on generating a particular output [@tag:Bahdanu2014_neural;
 @tag:Xu2015_show]. Deming et al. [@tag:Deming2016_genetic] applied the attention
 mechanism to models trained on genomic sequence. Attention mechanisms provide
 insight into the model's decision-making process by revealing which portions of
-the input are used by different outputs. In the clinical domain, Choi et al.
+the input are used by different outputs.
+Singh et al. uses a hierarchy of attention layers to locate important genome positions and signals for predicting gene expression from histone modifications [@tag:Singh2017_attentivechrome].
```
Comment (agitter) [inline content/06.discussion.md]: `uses` to `uses`.

File: content/citation-tags.tsv [hunk 9]
```diff
@@ -47,6 +47,7 @@ CudNN	arxiv:1410.0759
 Dahl2014_multi_qsar	arxiv:1406.1231
 Dean2012_nips_downpour	url:http://research.google.com/archive/large_deep_networks_nips2012.html
 DeepChem	url:https://github.com/deepchem/deepchem
+DeepSpeech	arxiv:1412.5567
```
Comment (agitter) [inline content/citation-tags.tsv]: This tag isn't used.  Can we remove it?  I think the paper was used in an edit to the introduction in #636, but it didn't look critical to me.

**Comment by dhimmel** [comment]
> If you agree with them, I can make the edits myself and merge.

@agitter, we should expedite this PR. I don't really think your suggestions need review, since they fall into the two categories:

1. not doing a certain change. Things that are not merged here can always be proposed in a future PR after the reflow.
2. obvious typo or grammer fixes.

This being said, I approve the changes that would result from your suggestion. If you add another commit, just make sure not to squash merge.

File: content/03.categorize.md (hunk 3 above)
Comment (qiyanjun) [reply content/03.categorize.md]: That would be nice! Most people working in this field call the training procedure using adversarial examples as "adversarial training". We can add both terms if possible.

File: content/03.categorize.md (hunk 2 above)
Comment (qiyanjun) [reply content/03.categorize.md]: Agree!

File: content/03.categorize.md (hunk 4 above)
Comment (qiyanjun) [reply content/03.categorize.md]: Agree!

File: content/04.study.md (hunk 5 above)
Comment (qiyanjun) [reply content/04.study.md]: Agree

File: content/04.study.md (hunk 6 above)
Comment (qiyanjun) [reply content/04.study.md]: Agree!

File: content/05.treat.md (hunk 7 above)
Comment (qiyanjun) [reply content/05.treat.md]: Sure.

File: content/06.discussion.md (hunk 8 above)
Comment (qiyanjun) [reply content/06.discussion.md]: Agree

File: content/citation-tags.tsv (hunk 9 above)
Comment (qiyanjun) [reply content/citation-tags.tsv]: Sure

**Comment by qiyanjun** [comment]
@agitter Please help me make the edits directly to the current master and as you prefer. Thanks

**Comment by agitter** [comment]
ad626d8 was a test commit to confirm I'm updating the right branch.  I'm making the rest of my proposed changes now.

**Comment by agitter** [comment]
@dhimmel I finished my edits and approved the pull request.  This is ready to merge.

**Comment by dhimmel** [comment]
@agitter mind if I squash this entire PR and making @qiyanjun the sole git author?

**Comment by agitter** [comment]
@dhimmel feel free to squash

**Pull request merged**